guide/clear/clear-words.md

```diff
@@ -22,24 +22,21 @@ Use words that are common and clear to as many people as possible.
 
 ## What To Do
 
-- Use at the most common words and phrases. These are the terms that people with lower language skills are most likely to know
+WCAG section 3.1 addresses several related aspects of content readability (see below). This includes:
```

Comment:
Moved to here because these WCAG criteria (and techniques) actually tell readers what to do

Reply:
+1